plugins/generator-1.17.1/forge-1.17.1/triggers/block_break.java.ftl

```diff
@@ -0,0 +1,19 @@
+@Mod.EventBusSubscriber private static class GlobalTrigger {
+	@SubscribeEvent public static void onBlockBreak(BlockEvent.BreakEvent event) {
+		Entity entity = event.getPlayer();
+		LevelAccessor world = event.getWorld();
+		Map<String, Object> dependencies = new HashMap<>();
+		dependencies.put("xpAmount",event.getExpToDrop());
+		dependencies.put("x",event.getPos().getX());
+		dependencies.put("y",event.getPos().getY());
+		dependencies.put("z",event.getPos().getZ());
+		dependencies.put("px",entity.getBlockX());

```

Comment:
getPosX was replaced with XXX and not getPosX, please fix these and check with forge-bot for the new naming to use the right name in the future.

![image](https://user-images.githubusercontent.com/16374228/131484048-3e5183e4-44b1-423b-addb-ac625df0d79f.png)